game-core/src/test/java/org/triplea/util/PointFileReaderWriterTest.java

```diff
@@ -600,39 +585,41 @@ void shouldWriteMultiplePlacementsPerName() throws Exception {
     void shouldNotCloseStream() throws Exception {
       final OutputStream os = newOutputStreamForCloseTest();
 
-      PointFileReaderWriter.writeOneToManyPolygons(os, Collections.emptyMap());
+      PointFileReaderWriter.writeOneToManyPolygons(os, Map.of());
 
       verify(os, never()).close();
     }
 
     @Test
     void shouldWriteMultiplePolygonsPerName() throws Exception {
       final Map<String, List<Polygon>> polygonListsByName =
-          ImmutableMap.of(
+          Map.of(
               "Belarus",
-                  Collections.singletonList(
-                      polygon(new Point(1011, 1021), new Point(1012, 1022), new Point(1013, 1023))),
+              List.of(polygon(new Point(1011, 1021), new Point(1012, 1022), new Point(1013, 1023))),
               "54 Sea Zone",
-                  Arrays.asList(
-                      polygon(new Point(2011, 2021), new Point(2012, 2022), new Point(2013, 2023)),
-                      polygon(new Point(2111, 2121), new Point(2112, 2122), new Point(2113, 2123))),
+              List.of(
+                  polygon(new Point(2011, 2021), new Point(2012, 2022), new Point(2013, 2023)),
+                  polygon(new Point(2111, 2121), new Point(2112, 2122), new Point(2113, 2123))),
               "Philippines",
-                  Arrays.asList(
-                      polygon(new Point(3011, 3021), new Point(3012, 3022), new Point(3013, 3023)),
-                      polygon(new Point(3111, 3121), new Point(3112, 3122)),
-                      polygon(new Point(3211, 3221))));
+              List.of(
+                  polygon(new Point(3011, 3021), new Point(3012, 3022), new Point(3013, 3023)),
+                  polygon(new Point(3111, 3121), new Point(3112, 3122)),
+                  polygon(new Point(3211, 3221))));
 
       final String content =
           writeToString(os -> PointFileReaderWriter.writeOneToManyPolygons(os, polygonListsByName));
 
+      assertThat(content, containsString("Belarus  <  (1011,1021)  (1012,1022)  (1013,1023)  > "));
+
       assertThat(
           content,
-          is(
-              ""
-                  + "Belarus  <  (1011,1021)  (1012,1022)  (1013,1023)  > \r\n"
-                  + "54 Sea Zone  <  (2011,2021)  (2012,2022)  (2013,2023)  > "
-                  + " <  (2111,2121)  (2112,2122)  (2113,2123)  > \r\n"
-                  + "Philippines  <  (3011,3021)  (3012,3022)  (3013,3023)  > "
+          containsString(
+              "54 Sea Zone  <  (2011,2021)  (2012,2022)  (2013,2023)  > "
+                  + " <  (2111,2121)  (2112,2122)  (2113,2123)  > "));
+      assertThat(
+          content,
+          containsString(
```

Comment:
Same here with line endings

Reply:
To be pedantic, there is a 3rd example as well. I don't think there is a need to point out every example once a general point has been made and it's clear that it's a general point that applies to every example that could be found.